### PR TITLE
Make Radars Great Again (MRGA)

### PR DESCRIPTION
**Pull request by TonybynMp4**
## Description

Radars weren't that useful :c
now they really are radars (even if the file is called ANPR wich isn't really a radar i guess but who cares)

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -531,6 +531,15 @@ QBCore.Functions.CreateCallback('police:server:IsPoliceForcePresent', function(_
 end)
 
 -- Events
+RegisterNetEvent('police:server:Radar', function(fine)
+    local source = source
+    local price  = Config.SpeedFines[fine].fine
+    local Player = QBCore.Functions.GetPlayer(source)
+    Player.Functions.RemoveMoney("bank", math.floor(price), "Radar Fine")
+    exports['qbx-management']:AddMoney('police', price)
```
Comment (FjamZoo) [inline server/main.lua:539]: At the VERY least, we need to make sure that the PLAYER MONEY, is removed before we add Police Money to management

Comment (TonybynMp4) [reply server/main.lua:539]: true

File: client/anpr.lua [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
+		if speed > v.maxspeed then
+			TriggerServerEvent('police:server:Radar', k)
+			TriggerServerEvent("InteractSound_SV:PlayOnSource", 'speedcamera', 0.25)
+			break
 		end
 	end
+end
+
+local function HandlespeedCam(speedCam, radar)
+	if cache.vehicle and cache.seat == -1 and GetVehicleClass(cache.vehicle) ~= 18 then
```
Comment (BerkieBb) [inline client/anpr.lua:17]: invert if statement to return instead of wrapping all code in an if statement

File: client/anpr.lua [hunk 3]
```diff
@@ -1,49 +1,52 @@
...
+	for _,value in pairs(Config.Radars) do
+		SpeedCams[#SpeedCams+1] = lib.points.new({
+			coords = vec3(value.coords.x, value.coords.y, value.coords.z),
+			distance = 20.0,
+			speed = value.speedlimit,
+		})
+	end
+	for _, v in pairs(SpeedCams) do
+		function v:onEnter()
+			HandlespeedCam(self, _)
```
Comment (BerkieBb) [inline client/anpr.lua:49]: an underscore is unusual, just make it k

File: client/anpr.lua [hunk 4]
```diff
@@ -1,49 +1,52 @@
...
-			for i = 1, #Config.Radars do
-				local value = Config.Radars[i]
-				HandleSpeedCam(value, i)
-			end
-			Wait(200)
+
+CreateThread(function()
+	for _,value in pairs(Config.Radars) do
+		SpeedCams[#SpeedCams+1] = lib.points.new({
+			coords = vec3(value.coords.x, value.coords.y, value.coords.z),
```
Comment (BerkieBb) [inline client/anpr.lua:42]: Instead of making a new vector3, you can also use `value.coords.xyz`, that returns the vector3 of the vector4

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client/anpr.lua (hunk 3 above)
Comment (TonybynMp4) [reply client/anpr.lua:49]: unusual ? it's in every resource

**Review by BerkieBb — APPROVED** [review]

**Comment by BerkieBb** [comment]
Now to avoid these, you just have to have 0 money

**Comment by TonybynMp4** [comment]
> Now to avoid these, you just have to have 0 money

we need Qbx debt
(i don't think the bank goes below 0?)

**Review by FjamZoo — APPROVED** [review]

**Pull request merged**